console-subscriber/src/lib.rs

```diff
@@ -962,6 +962,38 @@ impl Server {
         res?.map_err(Into::into)
     }
 
+    #[cfg(feature = "grpc-web")]
+    /// Starts the gRPC service with the default gRPC settings and gRPC-Web
+    /// support.
+    pub async fn serve_with_grpc_web(
```

Comment:
I think we should add an example here (see `into_parts` below) and a description of how this can be used to set CORS.

In fact, I think it would be good to include 2 examples, 1 basic one and 1 that is more complex (similar to or the same as the `grpc_web.rs` example) which shoulds how to set CORS middleware.

Reply:
Added. Thanks!

Reply:
<img width="1107" alt="image" src="https://github.com/tokio-rs/console/assets/29879298/7db01ca2-20a9-44ea-a751-b9de0d02bafc">